Test.C

```diff
@@ -1,53 +1,161 @@
+            {1, "Encryption1", "Encryption1_Description", 10, 0.75},
+            {2, "Encryption2", "Encryption2_Description", 10, 0.75},
+            {3, "Encryption3", "Encryption3_Description", 10, 0.75},
+            {4, "Encryption4", "Encryption4_Description", 10, 0.75},
+            {5, "Encryption5", "Encryption5_Description", 10, 0.75},
+            {6, "Encryption6", "Encryption6_Description", 10, 0.75},
+            {7, "Encryption7", "Encryption7_Description", 10, 0.75},
+            {8, "Encryption8", "Encryption8_Description", 10, 0.75},
+            {9, "Encryption9", "Encryption9_Description", 10, 0.75}
+    };
+
+    //initialize Attack structures
+    struct Attack attackStock[NUM_ATTACK] = {
+            {0, "Attack0", "Attack0_Description", 1},
+            {1, "Attack1", "Attack1_Description", 1},
+            {2, "Attack2", "Attack2_Description", 2},
+            {3, "Attack3", "Attack3_Description", 2},
+            {4, "Attack4", "Attack4_Description", 3}
+    };
 
-    //initialize player 
-
-    //create an equipment array of amount of equipment
-    //create a dymanic list from pointers to encrypt
-
-    //set startup equipment, encryption method, credits 
-
-     
-
-    while(player has options and more rounds exists){ 
-
-        //output player status (if statments)
+    //initialize 2d of attack effects (encrypt)
+    double attackEffectiveness[NUM_ENCRYPTION][NUM_ATTACK] = {
+            {/* Attack0: */ 0.1, /* Attack1 */ 0.3, /* Attack2: */ 0.8, /* Attack3: */ 1.0, /* Attack4: */ 1.0}, // Encryption0
+            {/* Attack0: */ 0.1, /* Attack1 */ 0.3, /* Attack2: */ 0.8, /* Attack3: */ 1.0, /* Attack4: */ 1.0}, // Encryption1
+            {/* Attack0: */ 0.1, /* Attack1 */ 0.3, /* Attack2: */ 0.8, /* Attack3: */ 1.0, /* Attack4: */ 1.0}, // Encryption2
+            {/* Attack0: */ 0.1, /* Attack1 */ 0.3, /* Attack2: */ 0.8, /* Attack3: */ 1.0, /* Attack4: */ 1.0}, // Encryption3
+            {/* Attack0: */ 0.1, /* Attack1 */ 0.3, /* Attack2: */ 0.8, /* Attack3: */ 1.0, /* Attack4: */ 1.0}, // Encryption4
+            {/* Attack0: */ 0.1, /* Attack1 */ 0.3, /* Attack2: */ 0.8, /* Attack3: */ 1.0, /* Attack4: */ 1.0}, // Encryption5
+            {/* Attack0: */ 0.1, /* Attack1 */ 0.3, /* Attack2: */ 0.8, /* Attack3: */ 1.0, /* Attack4: */ 1.0}, // Encryption6
+            {/* Attack0: */ 0.1, /* Attack1 */ 0.3, /* Attack2: */ 0.8, /* Attack3: */ 1.0, /* Attack4: */ 1.0}, // Encryption7
+            {/* Attack0: */ 0.1, /* Attack1 */ 0.3, /* Attack2: */ 0.8, /* Attack3: */ 1.0, /* Attack4: */ 1.0}, // Encryption8
+            {/* Attack0: */ 0.1, /* Attack1 */ 0.3, /* Attack2: */ 0.8, /* Attack3: */ 1.0, /* Attack4: */ 1.0} // Encryption9
+    };
+
+    //order attacks for game rounds
+    Round rounds[5] = {
+            {1, attackStock[0]},
+            {1, attackStock[0]},
+            {2, attackStock[0]},
+            {2, attackStock[0]},
+            {3, attackStock[0]},
```

Comment:
So whoever makes the function that assigns attacks to rounds would assign according to the attack tier set here.